app/src/main/java/de/westnordost/streetcomplete/quests/sidewalk/AddSidewalkForm.kt

```diff
@@ -1,11 +1,30 @@
 package de.westnordost.streetcomplete.quests.sidewalk
 
+import androidx.appcompat.app.AlertDialog
+import de.westnordost.streetcomplete.R
 import de.westnordost.streetcomplete.quests.AStreetSideSelectFragment
+import de.westnordost.streetcomplete.quests.AnswerItem
 
 class AddSidewalkForm : AStreetSideSelectFragment<Sidewalk, SidewalkSides>() {
 
     override val items = Sidewalk.values().map { it.asStreetSideItem() }
 
+    override val otherAnswers: List<AnswerItem> get() {
+        val isNoRoundabout = osmElement!!.tags["junction"] != "roundabout" && osmElement!!.tags["junction"] != "circular"
```

Comment:
```suggestion
```
Unused copy-paste relict?

Reply:
Or even better, replace the whole function with a static list:

```kt
    override val otherAnswers = listOf(
        AnswerItem(R.string.quest_sidewalk_answer_none) { noSidewalksHereHint() },
    )
```